test/test_xpu.py

```diff
@@ -428,6 +430,20 @@ def test_get_arch_list(self):
         for arch in arch_list:
             self.assertTrue(arch in flags)
 
+    def test_torch_version_xpu(self):
+        self.assertTrue(len(torch.version.xpu) == 8)
+        compiler_version = int(torch.version.xpu)
+        self.assertTrue(compiler_version >= 20230000)
+        if IS_LINUX:
```

Comment:
According to @atalman 's comments, I supplemented this UT to validate the build flow as we expected.

Reply:
Thank you